<launch>
    <arg name="debug" default = "0"/>
    <arg name="use_own" default="true"/>
    <group if="$(arg use_own)">
    <rosparam file="$(find ebobot)/config/nodes/planers.yaml" />
    </group>

    <node pkg="ebobot" type="global_planer.py" name="global_planer" output="screen" respawn="true" respawn_delay="1"> 
        <param name = "debug" value = "$(arg debug)"/>
    </node>

    <node pkg="ebobot" type="local_planer.py" name="local_planer" output="screen" respawn="true" respawn_delay="1">
        <param name = "debug" value = "$(arg debug)"/>
    </node>
</launch>